File: Data Structures & Algorithms/duplicate-integer/submission-5.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> map;
        for (auto it = nums.begin(); it != nums.end(); ++it){
            if (map.count(*it)){
                return true;
            }
            map.insert(*it);
            
        }
        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-5.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = {}
        res = []
        for num in nums:
            count[num] = 1 + count.get(num,0)
        arr = list(count.items())
        arr.sort(key=lambda x:x[1])
        while len(res) < k:
            res.append(arr.pop()[0])
        return res
        
        
        


            
            
        
        